spec/game_spec.rb

```diff
@@ -1,22 +1,36 @@
 require_relative '../game'
-require_relative 'mocks/mock_io'
+require_relative 'mocks/mock_io_handler'
 
 describe 'Game' do
-  
-  mock_io = MockIO.new
+
+  mock_io_handler = MockIOHandler.new
   prompter = Prompter.new
-  game = Game.new(prompter, mock_io)
+  game = Game.new(mock_io_handler, prompter)
 
   describe 'game_handler' do
 
-    it 'will receive printer from mock_view' do
-      expect(mock_io).to receive :printer
+    before do
+      mock_io_handler.word_blank(mock_io_handler.stubbed_value = "test")
+    end
+
+    after do
```

Comment:
thought this was a little weird at first -- since after is usually just used to tear things down. But it's a pretty cool idea :+1: so I'm in!